drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -202,30 +231,6 @@ public int getCopyTextColor() {
         return copyTextPaint.getColor();
     }
 
-    public void setText(String text) {
-        if (isSameText(text)) {
```

Comment:
`isSameText` was dropped as an unnecessary optimisation and makes the code easier to read when considering that the `setNumberOfDropCaps` also needs to call this method.